### PR TITLE
[improve][broker] Introduce `BitRateUnit` for calculate bitrate

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

Introduce `BitRateUnit` to avoid hard code in `LinuxInfoUtils`.

### Modifications

- Introduce `BitRateUnit`.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

- [x] `no-need-doc`

### CODE REVIEW

File: pulsar-common/src/main/java/org/apache/pulsar/common/util/BitRateUnit.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.util;
+
+public enum BitRateUnit {
```
Comment (merlimat) [inline pulsar-common/src/main/java/org/apache/pulsar/common/util/BitRateUnit.java:21]: If we're not going to need this in client lib, we should put this in `broker` or `broker-common` modules.

Comment (mattisonchao) [reply pulsar-common/src/main/java/org/apache/pulsar/common/util/BitRateUnit.java:21]: Ok, I will move it to `broker-common`

Comment (mattisonchao) [reply pulsar-common/src/main/java/org/apache/pulsar/common/util/BitRateUnit.java]: fixed, PTAL @merlimat

**Review by Jason918 — APPROVED** [review]
LGTM

**Review by gaozhangmin — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**